quinn-proto/src/connection/spaces.rs

```diff
@@ -233,6 +246,7 @@ pub struct Retransmits {
     pub(super) crypto: VecDeque<frame::Crypto>,
     pub(super) new_cids: Vec<IssuedCid>,
     pub(super) retire_cids: Vec<u64>,
+    pub(super) ack_frequency: Option<u64>,
```

Comment:
I think this should be ok, as it is explicitly contemplated in the draft (the comment in brackets is mine):

> When an ACK_FREQUENCY frame is lost, the sender is encouraged to send another ACK_FREQUENCY frame [instead of resending the original frame].

I'll update the code